notebooks/3_subset_data.ipynb

```diff
@@ -0,0 +1,1358 @@
+{
```

Comment:
<p>This seems like a great place to introduce the shape attribute. You can show here that a Series is 1-dimensional, and you can contrast that below with the DataFrame being 2-dimensional. Introducing shape here will also help later in this lesson.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stijnvanhoey/pandas-getting-started-tutorials/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>